library/core/registry/src/main/java/org/quiltmc/qsl/registry/impl/sync/server/ServerRegistrySync.java

```diff
@@ -0,0 +1,349 @@
+/*
+ * Copyright 2022-2023 QuiltMC
```

Comment:
Is this a new class or a class move?

First case: copyright year should be this year.
Second case: copyright year should be 2022.

Reply:
Just class move